qmarkdowntextedit.cpp

```diff
@@ -340,7 +340,7 @@ bool QMarkdownTextEdit::eventFilter(QObject *obj, QEvent *event) {
                     setTextCursor(cursor);
                 }
             }
-            return false;
+            return QPlainTextEdit::eventFilter(obj, event);
```

Comment:
Wouldn't those changes change the behavior?

Reply:
@Waqar144, what's your opinion?

Reply:
would be more or less the same I think.

What would be better here overall would be that the entire eventFilter is removed and events are handled in event Functions for e.g., keyEvent() etc. Would clean things up a bit.

Reply:
I see. Thank you. I'll merge then in the meanwhile.